src/javaRosa/itextWidget.js

```diff
@@ -305,6 +305,7 @@ define([
                 if (e.form === widget.form && e.itextType === widget.itextType) {
                     if (widget.getItextValue() === e.prevValue) {
                         // Make sure all the defaults keep in sync.
+                        // why doesn't setValue set the itext value?
```

Comment:
This does seem weird, moreso after reading through these changes and seeing the various places where you have to call both `setValue` and `setItextValue` together. It seems odd that `setItextValue` is publicly exposed. Do tests fail if you move `setItextValue` inside `setValue`?